mobile/src/components/uam/groups-table/groups-table.tsx

```diff
@@ -13,7 +19,8 @@ import { styles } from './styles';
 const GroupsTable: FC = () => {
   const dispatch = useAppDispatch();
   const { width } = useWindowDimensions();
-  const { items } = useAppSelector((state) => state.uam.groups);
+  const { items, total } = useAppSelector((state) => state.uam.groups);
+  const [page, setPage] = useState<number>(PaginationDefaultValue.DEFAULT_PAGE);
```

Comment:
Create the hook `usePagination`(take a look to frontend part)

Reply:
I need to install react-router-native for useSearchParams hook or just do it via useState ?

Reply:
We don't need to handle search params in mobile app